stl/inc/algorithm

```diff
@@ -690,30 +690,37 @@ _NODISCARD pair<_FwdIt1, _FwdIt2> mismatch(
 
 #ifdef __cpp_lib_concepts
 namespace ranges {
-    class _Equal_fn : private _Not_quite_object {
-    private:
-        template <class _It1, class _It2, class _Size, class _Pr, class _Pj1, class _Pj2>
-        _NODISCARD static constexpr bool _Equal_count(
-            _It1 _First1, _It2 _First2, _Size _Count, _Pr _Pred, _Pj1 _Proj1, _Pj2 _Proj2) {
-            if constexpr (_Equal_memcmp_is_safe<_It1, _It2,
-                              _Pr> && same_as<_Pj1, identity> && same_as<_Pj2, identity>) {
-                if (!_STD is_constant_evaluated()) {
-                    return _Memcmp_count(_First1, _First2, static_cast<size_t>(_Count)) == 0;
-                }
+    // clang-format off
+    template <input_iterator _It1, input_iterator _It2, integral _Size, class _Pr, class _Pj1, class _Pj2>
+        requires indirectly_comparable<_It1, _It2, _Pr, _Pj1, _Pj2>
+    _NODISCARD static constexpr bool _Equal_count(

```

Comment:
As `_Equal_count()` is being extracted from class scope to namespace scope, it should no longer be `static`.